package.json

```
@@ -1,12 +1,14 @@
 {
-  "name": "cld3",
+  "name": "node-cld3",
```

```suggestion
  "name": "cld3",
```
https://www.npmjs.com/package/cld3 <- we already took the package name here